### PR TITLE
Specify QDROUTE image for Intero tests

### PR DESCRIPTION
**Pull request by granzoto**
@fgiorgetti  @nicob87 could you please review ?
Signed-off-by: Renato Granzoto <rgranzot@redhat.com>

### CODE REVIEW

File: pkg/topologies/smokev1/smokev1.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/rh-messaging/shipshape/pkg/apps/qdrouterd/qdrmanagement/entities"
 	"github.com/rh-messaging/shipshape/pkg/framework"
 	"github.com/rh-messaging/shipshape/pkg/framework/log"
+    "os"
```
Comment (fgiorgetti) [inline pkg/topologies/smokev1/smokev1.go:14]: @granzoto please fix the format here.

File: pkg/topologies/smokev1/smokev1.go [hunk 2]
```diff
@@ -184,6 +185,9 @@ func (s *SmokeRouterOnlyTopology) initializeInteriors() {
 
 	// Initializing the interior routers
 	s.IcInteriorEast = s.defaultInteriorSpec()
+    if os.Getenv("IMAGE_QDROUTERD_INTEROP") != "" {
+        s.IcInteriorEast.DeploymentPlan.Image = os.Getenv("IMAGE_QDROUTERD_INTEROP")
+    }
```
Comment (fgiorgetti) [inline pkg/topologies/smokev1/smokev1.go:190]: Same here... just running a `go fmt`, should help, I believe.

**Review by fgiorgetti — CHANGES_REQUESTED** [review]
Changes look good. Please just adjust the formatting.

File: pkg/topologies/smokev1/smokev1.go (hunk 1 above)
Comment (granzoto) [reply pkg/topologies/smokev1/smokev1.go]: fixed

File: pkg/topologies/smokev1/smokev1.go (hunk 2 above)
Comment (granzoto) [reply pkg/topologies/smokev1/smokev1.go]: fixed

**Review by fgiorgetti — APPROVED** [review]

**Pull request merged**